docs/DeveloperGuide.md

```diff
@@ -273,16 +273,16 @@ The target users...
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
-
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+Priorities: Essential (must have) - `* * *`, Typical (nice to have) - `* *`, Novel (unlikely to have) - `*`
+
+| Priority | As a …​ | I want to …​                                                         | So that I can…​                                                       |
+|----------|--------|----------------------------------------------------------------------|-----------------------------------------------------------------------|
+| `* * *`  | new GP | see usage instructions                                               | refer to instructions when I forget how to use the App                |
+| `* * *`  | GP     | deal with patient and specialist records separately                  | easily do patient-specific or specialist-specific operations          |
+| `* * *`  | GP     | access a patient's age                                               | provide age-specific care and treatment to the patient                |
+| `* *`    | GP     | filter patient records by criteria such as medical condition and age | simplify the process of finding specific patient information          |
+| `*`      | GP     | customise tags for my patients                                       | have control over the organisation of my patients                     |
+| `*`      | GP     | group delete patients by category or tag                             | save time when having to delete a large chunk of patients             |
```

Comment:
How about:
`having to delete a large chunk of patients` → `removing large amount of patient records` ?

Reply:
Thanks! I have made the requested changes.